### PR TITLE
latest motr di tests and lib updates - s76

### PR DESCRIPTION
**Pull request by pranav-sahasrabudhe**
Signed-off-by: Pranav Sahasrabudhe <Pranav.p.Sahasrabudhe@seagate.com>

Signed-off-by: pranav-sahasrabudhe <pranav.p.sahasrabudhe@seagate.com>

# Problem Statement
- Problem statement

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [ ] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: comptests/motr/test_motr_di.py [hunk 1]
```diff
@@ -95,13 +99,44 @@ class TestCorruptDataDetection:
...
+        for node in CMN_CFG["nodes"]:
+            node_obj = LogicalNode(
+                hostname=node["hostname"], username=node["username"], password=node["password"]
+            )
+            if node["node_type"].lower() == "master":
+                cls.master_node_list.append(node_obj)
+            else:
+                cls.worker_node_list.append(node_obj)
+
+            # cls.health_obj = Health(
```
Comment (pragamj) [inline comptests/motr/test_motr_di.py:131]: can we remove the commented code

Comment (pranav-sahasrabudhe) [reply comptests/motr/test_motr_di.py]: Removed

File: comptests/motr/test_motr_di.py [hunk 2]
```diff
@@ -210,11 +389,87 @@ def test_m0cp_m0cat_block_corruption_unaligned(self):
...
+        and then follow next steps:
+        Copy motr object with m0cp
+        Identify parity block using m0trace logs created during m0cp
+        Corrupt parity block using m0cp+error_injection.py script
+        Read from object with m0cat should throw an error.
+        -s 4096 -c 10 -o 1048583 /root/infile -L 3
+        -s 4096 -c 1 -o 1048583 /root/myfile -L 3 -u -O 0
+        -o 1048583 -s 4096 -c 10 -L 3 /root/dest_myfile
+        """
+        count_list = [["4", "1"]]
```
Comment (pragamj) [inline comptests/motr/test_motr_di.py:416]: we can send only ['4'] as we are not using cp_update to corrupt.

**Review by pragamj — APPROVED** [review]
Requested few changes.

**Review by ravindrachoudhari — APPROVED** [review]

**Review by ketanarlulkar — APPROVED** [review]

File: comptests/motr/test_motr_di.py (hunk 2 above)
Comment (pranav-sahasrabudhe) [reply comptests/motr/test_motr_di.py:416]: As discussed will take this later as it might need bigger fix for trivial change.

**Review by ravindrachoudhari — APPROVED** [review]

**Pull request merged**